From 8f8698aba19b501f01285e9eec5c18231fc6bcea Mon Sep 17 00:00:00 2001
From: Vratislav Podzimek <vpodzime@redhat.com>
Date: Tue, 6 Jan 2015 13:06:52 +0100
Subject: Do not include config.h in libvolume_key.h

The library's header file distributed in the devel package cannot include
the config.h file that is only available during build otherwise it's not
possible to use the library outside of the volume_key build process.

Signed-off-by: Vratislav Podzimek <vpodzime@redhat.com>

diff --git a/lib/libvolume_key.h b/lib/libvolume_key.h
index 657b626..513f923 100644
--- a/lib/libvolume_key.h
+++ b/lib/libvolume_key.h
@@ -18,8 +18,6 @@ Author: Miloslav Trmaƒç <mitr@redhat.com> */
 #ifndef LIBVOLUME_KEY_H__
 #define LIBVOLUME_KEY_H__
 
-#include <config.h>
-
 #include <cert.h>
 #include <glib.h>
 
-- 
cgit v0.10.2

